features/mfa.feature

```diff
@@ -137,13 +141,13 @@ Feature: Prompt for MFA credentials
       |           |   TOTP   |                | supports WebAuthn         |   TOTP           |
       |           |   TOTP   | , backup codes | supports WebAuthn         |   TOTP           |
       |           |          |   backup codes | supports WebAuthn         |      backup code |
-      | WebAuthn  |          |                | does not support WebAuthn | WebAuthn         |
-      | WebAuthn  | , TOTP   |                | does not support WebAuthn |   TOTP           |
-      | WebAuthn  |          | , backup codes | does not support WebAuthn |      backup code |
-      | WebAuthn  | , TOTP   | , backup codes | does not support WebAuthn |   TOTP           |
-      |           |   TOTP   |                | does not support WebAuthn |   TOTP           |
-      |           |   TOTP   | , backup codes | does not support WebAuthn |   TOTP           |
-      |           |          |   backup codes | does not support WebAuthn |      backup code |
+#      | WebAuthn  |          |                | does not support WebAuthn | WebAuthn         |
+#      | WebAuthn  | , TOTP   |                | does not support WebAuthn |   TOTP           |
+#      | WebAuthn  |          | , backup codes | does not support WebAuthn |      backup code |
+#      | WebAuthn  | , TOTP   | , backup codes | does not support WebAuthn |   TOTP           |
+#      |           |   TOTP   |                | does not support WebAuthn |   TOTP           |
+#      |           |   TOTP   | , backup codes | does not support WebAuthn |   TOTP           |
+#      |           |          |   backup codes | does not support WebAuthn |      backup code |
```

Comment:
This seems related to my previous two comments. Is lack-of-WebAuthn-support something that's (essentially) not a thing any more?